doc/single-nic-vm-host.md

```diff
@@ -0,0 +1,194 @@
+# How to configure a virtual machine host with a single network interface
+
+In this How to, you will learn how to configure a virtual machine host using netplan and virsh. The host in this scenario has a single network interface. 
+
+
+## Prerequisites
+
+Before we can get started, we need to establish our setup and make sure our prerequisite steps are completed and tested.
+
+### Reference setup
+
+- A computer with a single NIC.
+- Ubuntu Server installed.
+- QEMU/KVM installed.
+- IPv4: 
+    - Network: 192.168.150.0/24
+    - DNS1: 1.1.1.1
+    - DNS2: 8.8.8.8
+- A switch
+- A router/firewall
+    - IPv4: 192.168.150.254/24
+    - Firewall policies, DNS, DHCP configured and tested.
+
+
+### Elevated privileges
+All commands in this tutorial assume you are using elevated privileges.
+
+### QEMU/KVM installed
+QEMU/KVM should be installed. Installation instructions can be found on [this Ubuntu Wiki page](https://help.ubuntu.com/community/KVM/Installation).
+
+### Disable netfilter for bridged interfaces
+
+We need to disable netfilter for bridged interfaces, in order to allow communications between the homeserver, its virtual machines, and the devices in the local VLANs.
+
+- Edit this file: ```/etc/systemctl.conf```
+- Add the following lines to the file
+
+```
+net.bridge.bridge-nf-call-iptables = 0
+net.bridge.bridge-nf-call-ip6tables = 0
+net.bridge.bridge-nf-call-arptables = 0
+```
+
+- Apply the changes immediately, without rebooting the host.
+
+```
+sysctl -p /etc/sysctl.conf
+```
+
+
+### Edit or firewall rules
+
+
+Remember to edit, configure, or disable firewall according to your needs. You can refer to [ufw documentation](https://help.ubuntu.com/community/UFW) if you need help configuring it.
+
+
+
+## Netplan configuration
+
+
+
+### Edit netplan configuration
+
+Edit netplan configuration file:  ```/etc/netplan/00-installer-config.yaml```
```

Comment:
```suggestion
Edit Netplan's configuration file:  ```/etc/netplan/00-installer-config.yaml```
```

Reply:
A note on this: the possessive here is not necessary ("Netplan configuration file" works just as well), and we try to avoid anthropomorphizing products.